packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts

```diff
@@ -0,0 +1,48 @@
+// @group:wallet
+// @retry=2
+
+import { NetworkSymbol } from 'suite-common/wallet-config/src';
+import { onAccountsPage } from '../../support/pageObjects/accountsObject';
+
+describe('Check coins XPUB', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true });
+        cy.task('setupEmu', {
+            needs_backup: false,
+            mnemonic: 'town grace cat forest dress dust trick practice hair survey pupil regular',
+        });
+        cy.task('startBridge');
+
+        cy.viewport(1080, 1440).resetDb();
+        cy.prefixedVisit('/accounts');
+        cy.passThroughInitialRun();
+        cy.discoveryShouldFinish();
+    });
+
+    afterEach(() => {
+        cy.task('stopEmu');
+    });
+
+    const coins: NetworkSymbol[] = ['btc', 'ltc', 'vtc', 'ada'];
+
+    /**
+     * 1. Start at accounts
+     * 2. If not Bitcoin activate coin
+     * 3. apply coin filter
+     * 4. check xpub
+     * 5. x
+     */
```

Comment:
Update test case documentation and remove blank line on 35

Reply:
done